### DISCUSSION TITLE
开启插件后会，会出现人机验证不通过的情况

### DISCUSSION THREAD

**Discussion by coolinmind** [anghunk/linuxdo-scripts#22] [General]
linux do有时候会出发cloudflare真人验证，插件开启状态就会验证不通过，关闭插件后刷新，就能正常通过，请求修复，谢谢！

**Comment by anghunk** [comment]
@coolinmind 有什么前置条件吗，比如未登录，开启代理之类的

按理说插件跟人机验证无关，我得等遇到了才能知道问题出在哪

**Comment by coolinmind** [comment]
登录状态，未开启代理，linux do页面挂起久了就需要刷新页面才能正常浏览，此过程就会触发人机验证。此插件开启状态，验证就会反复循环，无法正常进入首页，关闭插件后才能正常进入。

**Comment by anghunk** [comment]
> 登录状态，未开启代理，linux do页面挂起久了就需要刷新页面才能正常浏览，此过程就会触发人机验证。此插件开启状态，验证就会反复循环，无法正常进入首页，关闭插件后才能正常进入。

先mark一下，等我遇到了这个情况才能知道问题出在哪，或者等其他佬反馈一下具体情况，现在分析不出来 😂

**Comment by anghunk** [comment]
重构后的新版本是否还存在该问题。

**Comment by CangWeiohh** [comment]
> 重构后的新版本是否还存在该问题。

问题依然存在

版本:
0.3.11

可能的复现条件:
linuxdo网页标签挂着但没打开, 大概十几分钟左右, 再打开这个标签去刷新, 就会出现人机验证, 开着插件就会反复验证

**Comment by anghunk** [comment]
> > 重构后的新版本是否还存在该问题。
> 
> 问题依然存在
> 
> 版本: 0.3.11
> 
> 可能的复现条件: linuxdo网页标签挂着但没打开, 大概十几分钟左右, 再打开这个标签去刷新, 就会出现人机验证, 开着插件就会反复验证

好奇怪，是不是代理的问题啊，我这边还没遇到过，标签页一直开着

**Comment by CangWeiohh** [comment]
> > > 重构后的新版本是否还存在该问题。
> > 
> > 
> > 问题依然存在
> > 版本: 0.3.11
> > 可能的复现条件: linuxdo网页标签挂着但没打开, 大概十几分钟左右, 再打开这个标签去刷新, 就会出现人机验证, 开着插件就会反复验证
> 
> 好奇怪，是不是代理的问题啊，我这边还没遇到过，标签页一直开着

不清楚是不是代理的问题, 但是没有关闭或者重启代理的情况下, 关掉linuxdo插件, 就能过人机验证了

**Comment by anghunk** [comment]
@CangWeiohh 
测试出来，我切成成弱网环境，代理延迟2000+的时候会频繁触发人机验证
换成速度快的代理，会直接进论坛无需人机验证
个人感觉跟插件的关系不是很大（因为不知道从何下手，也有可能是油猴与Discourse之前在延迟大的情况下的冲突）

**Comment by CangWeiohh** [comment]
> @CangWeiohh 测试出来，我切成成弱网环境，代理延迟2000+的时候会频繁触发人机验证 换成速度快的代理，会直接进论坛无需人机验证 个人感觉跟插件的关系不是很大（因为不知道从何下手，也有可能是油猴与Discourse之前在延迟大的情况下的冲突）

暂时没有再出现人机验证了
